src/main/python/camdkit/model.py

```diff
@@ -194,6 +192,60 @@ def make_json_schema() -> dict:
     }
 
 
+class Transforms(TransformsParameter):
+  """
+  X,Y,Z in metres of camera sensor relative to stage origin.
+  The Z axis points upwards and the coordinate system is right-handed.
+  Y points in the forward camera direction (when pan, tilt and roll are zero).
+  For example in an LED volume Y would point towards the centre of the LED wall and so X would point to camera-right.
+  Rotation expressed as euler angles in degrees of the camera sensor relative to stage origin
+  Rotations are intrinsic and are measured around the axes ZXY, commonly referred to as [pan, tilt, roll]
+  Notes on Euler angles:
+  Euler angles are human readable and unlike quarternions, provide the ability for cycles (with angles >360 or <0 degrees).
+  Where a tracking system is providing the pose of a virtual camera, gimbal lock does not present the physical challenges of a robotic system.
+  Conversion to and from quarternions is trivial with an acceptable loss of precision
+  """
+  canonical_name = "transforms"
+  units = "metres / degrees"
+
+class TimingMode(TimingModeParameter):
+  """
+  'external' timing mode describes the case where the transport packet has inherent timing, so no explicit timing data is required in the data).
```

Comment:
This is not needed: either a PTP timestamp is present or it is not.

Reply:
The idea of this was that you can still use PTP timing even if the transport you are in doesn't have it inherently. Can you think of a better name? Intrinsic / extrinsic?

Reply:
Why is knowing the source of the PTP timestamp important?